test/features/step-definitions/steps.js

```diff
@@ -0,0 +1,62 @@
+const chai = require("chai");
+const { Given, When, Then } = require("cucumber");
+const addComponentPage = require("../pageobjects/add-component.page");
+const ConfigPage = require("../pageobjects/config.page");
+const EditPageSection = require("../pageobjects/edit-page.section");
+const FormDesignerPage = require("../pageobjects/form-designer.page");
+const MenuSection = require("../pageobjects/menu.section");
+
+const pages = {
+  start: ConfigPage,
+};
+
+Given("I have created a new form configuration", () => {
+  pages["start"].open();
+  var configRef = `smoke-testing ${Date.parse(Date())}`;
+  ConfigPage.newConfig(configRef);
+  expect(browser).toHaveUrlContaining(configRef.replace(" ", "-"));
+});
+
+When("I add a {string} control to the {string}", (componentName, pageName) => {
+  this.pageName = pageName
+  FormDesignerPage.createComponentForPageName(pageName).click();
+  addComponentPage.selectComponentByName(componentName);
+  addComponentPage.completeDateField(
+    "Date of Birth",
+    "dateOfBirth",
+    "Please enter your date of birth using the format dd/mm/yyyy"
+  );
+});
+
+Then("the Date field control is displayed in the page", () => {
+  chai.expect(FormDesignerPage.dropdown(this.pageName).isDisplayed()).to.be.true;
```

Comment:
did you mean to use two different expects? you can just do `const { expect } = require("chai")` so you don't have to do `chai.`

Reply:
So sneakily expect is built into webdriverio by default and has a bunch of default matchers, however I felt special enough to use chai in certain cases :)